spec.bs

```diff
@@ -2952,6 +2960,189 @@ This specification defines two [=policy-controlled features=] identified by the
+:: Null or a [=string=].
+  Opaque JSON data passed to the seller's [=script runner=].
+: <dfn>per buyer signals</dfn>
+:: A [=map=] whose [=map/keys=] are [=origins=] and whose [=map/values=] are [=strings=].
+  [=map/Keys=] are buyers and must be valid HTTPS origins. [=map/Values=] are opaque JSON data
+  passed to corresponding buyer's [=script runner=].
+
+</dl>
+
+Each [=traversable navigable=] has a <dfn for="traversable navigable">captured ad auction headers
+</dfn>, which is a [=map=] whose [=map/keys=] are [=direct from seller signals keys=] and whose
+[=map/values=] are [=direct from seller signals=].
+</div>
+
+<div algorithm="fetch auction headers patch">
+The following will be added to [[Fetch#http-extensions]]:
+
+<h3 id=sec-ad-auction-fetch-header>The \`<a http-header><code>Sec-Ad-Auction-Fetch</code></a>\` HTTP
+request header</h3>
+
+The \`<dfn http-header><code>Sec-Ad-Auction-Fetch</code></dfn>\` request header is an optional
+[=structured header=] with of type [=structured header/boolean=]. If equal to `?1`, the user agent
+will remove any [:Ad-Auction-Signals:] from the response returned to
+{{WindowOrWorkerGlobalScope/fetch()}} -- the [:Ad-Auction-Signals:] value will instead only be used
+in Protected Audiences auctions. [:Sec-Ad-Auction-Fetch:] will only be set if the
+{{WindowOrWorkerGlobalScope/fetch()}} call was made with the `{adAuctionHeaders: true}` option.
+
+<h3 id=ad-auction-signals-header>The \`<a http-header><code>Ad-Auction-Signals</code></a>\` HTTP
+response header</h3>
+
+The \`<dfn http-header><code>Ad-Auction-Signals</code></dfn>\` response header provides value
+of a JSON array of dictionaries, each with an `adSlot` key. Protected Audience non-component,
+component, and top-level auctions may specify which signals to load by the `adSlot` key.
+</div>
+
+<div algorithm="fetch auction signals response patch">
+The following step will be added to the [=HTTP-network-or-cache fetch=] algorithm, before step
+"Return |response|.":
+
+1. If |response| is not null, and |fetchParams|'s [=fetch params/task destination=] is a [=global
+  object=], then run [=update captured headers=] with |fetchParams|'s [=fetch params/task
+  destination=]'s [=associated Document's=] [=node navigable's=] [=traversable navigable's=]
+  [=traversable navigable/captured ad auction headers=], |response|'s [=response/header list=], and
+  <var ignore>request</var>'s [=request/URL=]'s [=url/origin=].
+
+</div>
+
+<div algorithm="fetch update captured headers patch">
+The following algorithm will be added to the [[FETCH#fetching]] section:
+
+<h3 id=update-captured-headers>Update captured headers</h3>
+
+  To <dfn id=concept-update-captured-headers>update captured headers</dfn> with a [=traversable
+  navigable/captured ad auction headers=] |storedHeaders|, [=header list=] |responseHeaders|, and
+  [=origin=] |requestOrigin|:
+  1. Let |adAuctionSignals| be the result of [=header list/get=] with [:Ad-Auction-Signals:],

```

Comment:
This reads a bit weird. We should write this as https://fetch.spec.whatwg.org/#ref-for-concept-header-list-get%E2%91%A3, for example.